### ISSUE TITLE
Create a project skeleton

### ISSUE THREAD

**Issue by atik-lab** [prismaproject/prismaproject#2]
Well here we should discuss how we start the project. IMO it will be easier to start from 0 and the copy parts form the former project as we develop than to start with the old project and have to check very carefully the use of the older algorithm.

In addition, I would like to discuss if we continue developing in Python or we can jump directly to a Go or Rust language that we will do anyway in the end.

**Comment by joeybe** [comment]
We dont need to start from scratch. The basics of networking and crypto libs are there. New database and consensus is what we need to recode.

**Comment by atik-lab** [comment]
Yes but rewriting the database and the consensus is 50% of the project, or more if we include the work of the signed state.

I just wanted to put on the table the possibility to code it in golang, a language that performs a thousand times better than python and uses multithreading (other big projects use golang, like sia and ethereum), i think that is better now than to have to rewrite all the code later and then we have to maintain 2 versions. Is this discarded?
